relay-server/src/actors/processor.rs

```diff
@@ -1261,13 +1281,49 @@ impl EnvelopeProcessorService {
         }
     }
 
+    fn check_cardinality_limits(
+        &self,
+        enable_cardinality_limiter: bool,
+        _organization_id: u64,
```

Comment:
nit: Should make this function `#[cfg(feature = "processing")]`?

```suggestion
        organization_id: u64,
```

Reply:
Unfortunately this is needed for non-processing, the entire function just exists to handle the `if` case with an early return to make the code somewhat readable and not a pure if else cfg mess.

Reply:
Is it still a mess with the `if_processing!` macro?